tests/test_identifier.py

```diff
@@ -1,8 +1,28 @@
+import json
+import re
+
 from pywhat import identifier
 from pywhat.distribution import Distribution
 from pywhat.helper import Keys
 
 
+def test_check_keys_in_json():
+    with open("pywhat/Data/regex.json", "r") as file:

```

Comment:
We have helper function for that, could you use it?

Reply:
Oh I didn't know about that! It uses the helper function now. Thank you :D